src/arrayMethodSort.js

```diff
@@ -1,11 +1,18 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(
+    compareFunction = (a, b) => a.toString() > b.toString()
```

Comment:
The default compare function should return a number, not a boolean. It should return a negative, zero, or positive value depending on the arguments.